src/js/vendors-layer.js

```diff
@@ -22,7 +22,31 @@ const vendorsSource = new VectorSource({
 
 const vendorsLayer = new VectorLayer({
   source: vendorsSource,
-  style: vendorsStyle
+  style: function (feature) {
+    if (feature.values_.pos_agent === 'FALSE') {
+      return new Style({
+        image: new Circle({
+          radius: 7,
+          fill: new Fill({ color: [240, 81, 89, 0.5] }),
+          stroke: new Stroke({
+            color: [140, 35, 40, 0.5],
+            width: 2
+          })
+        })
+      })
+    } else {
+      return new Style({
+        image: new Circle({
+          radius: 7,
+          fill: new Fill({ color: [90, 100, 125, 0.5] }),
+          stroke: new Stroke({
+            color: [53, 85, 166, 0.5],
+            width: 2
+          })
+        })
+      })
+    }
+  }
```

Comment:
@iholl, maybe good practice to think about how you might encapsulate this into a function? Hint, the function will take `feature` as an argument.